MediaGalleryUi/view/adminhtml/web/js/sorting.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'uiElement',
+    'uiRegistry'
+], function (Element, uiRegistry) {
+    'use strict';
+
+    return Element.extend({
+        defaults: {
+            template: 'Magento_AdobeStockImageAdminUi/sorting',
```

Comment:
media gallery should not depend on Magento_AdobeStockImageAdminUi, we have to either copy the template or move it to Magento_Ui module to use in both modules.